instance/models/mixins/openedx_config.py

```diff
@@ -383,12 +384,20 @@ def _get_configuration_variables(self):
             })
 
         if self.admin_users:
+            session = retry_session()
+            users = [
+                username for username in self.admin_users
+                if session.get('https://github.com/{}.keys'.format(username)).status_code == 200
+            ]
             template.update({
                 "COMMON_USER_INFO": [
                     {
                         "name": github_username,
                         "github": True,
                         "type": "admin"
+                    } if github_username in users else {
```

Comment:
@viadanna, do we have to include the users with missing keys as they will not have a way to login?

Reply:
I thought it was better than just ignoring the user. I can remove this instead.